docs/openapi/openapi.yml

```diff
@@ -1,51 +0,0 @@
-openapi: 3.1.0
```

Comment:
I don't think we should blast this.

The `stellar-turrets-docs` expects a correct `openapi.yml` file that this docs folder generates / keeps up to date

Reply:
Note - this should be blasted. Looks like docs repo will be the home for everything to avoid confusion.